### ISSUE TITLE
Deprecation warning using Week primitive with pandas 1.1.0 installed

### ISSUE THREAD

**Issue by thehomebrewnerd** [FeatureLabs/featuretools#1091]
In panadas version `1.1.0`, `Series.dt.weekofyear` and `Series.dt.week` have been deprecated. This results in a warning when using the `Week` transform primitive.

This primitive should be updated to prevent this warning. After updating, compatibility should be checked with older versions of pandas as well to determine if the pandas version specified in `requirements.txt` needs to be updated.

```
FutureWarning: Series.dt.weekofyear and Series.dt.week have been deprecated.  Please use Series.dt.isocalendar().week instead.
```

[Pandas documentation](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.dt.week.html)

**Comment by thehomebrewnerd** [comment]
Additional testing shows that the fix suggested in the deprecation warning only work with pandas `1.1.0` and nothing older, so fixing this issue would require setting the minimum pandas version to `>=1.1.0` in `requirements.txt`.

**Comment by thehomebrewnerd** [comment]
Closed by PR #1094 

Future updates to this should be tracked in Issue #1096
